admin/controllers/service-manager.js

```diff
@@ -81,11 +80,11 @@ const setUpdateTiming = async (req, res, next) => {
   res.locals.pageTitle = 'Check settings'
   let settings
 
-  const settingsRecord = await Settings.findOne().exec()
+  const settingsRecord = await settingDataService.findOne()
   if (settingsRecord) {
     settings = settingsRecord
   } else {
-    settings = new Settings()
+    settings = {}
```

Comment:
same as the settingsLog is this going to matter if it's an empty object vs undefined?

Reply:
I guess it does cause the query will thrown the exception.
Nevermind this comment

Reply:
Well it gets set later on...